doc/guide/readme.md

```diff
@@ -66,36 +66,3 @@ To get to the repl:
 
 docker run -it gerbil/scheme:latest /root/gerbil/bin/gxi
 ```
```

Comment:
Isn't this ``` extraneous? Shouldn't there be a link to the `Dockerfile` in case someone wants to layer on it, etc.?

Reply:
There is a missing one above. 
I removed the dockerfile as it was not exactly correct, but we should have a Dockerfile in tree (probably another pr).